content/en/docs/reference/setup-tools/kubeadm/kubeadm-reset.md

```diff
@@ -34,6 +34,36 @@ etcdctl del "" --prefix
 
 See the [etcd documentation](https://github.com/coreos/etcd/tree/master/etcdctl) for more information.
 
+### CNI configuration cleanup
+
+CNI plugins use the directory `/etc/cni/net.d` to store their configuration.
+The `kubeadm reset` command does not cleanup that directory. Backup the directory if needed.
+Then to remove it execute the following command:
+
+```bash
+sudo rm -rf /etc/cni/net.d
+```
+
+### Network traffic rules cleanup
+
+The `kubeadm reset` command does not clean any iptables, nftables or IPVS rules applied
+to the host by kube-proxy. If you wish to perform this cleanup, you can use the same
+kube-proxy container which was used in your cluster and the `--cleanup` flag of the
+kube-proxy binary:
+
+```bash
+docker run --privileged registry.k8s.io/kube-proxy:v{{< skew currentPatchVersion >}} sh -c "kube-proxy --cleanup && echo DONE"
```

Comment:
It is better to add the `--rm` flag. It will automatically remove the container when it exits.